src/amber/cli/templates/app/config/deploy/Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM drujensen/crystal:0.22.0-1
+FROM drujensen/crystal:0.23.1
 
-ENV AMBER_VERSION 0.1.15
+ENV AMBER_VERSION 0.2.3
 
-RUN curl -L https://github.com/Amber-Crystal/amber_cmd/archive/v$AMBER_VERSION.tar.gz | tar xvz -C /usr/local/share/. && cd /usr/local/share/amber_cmd-$AMBER_VERSION && crystal deps && make
+RUN curl -L https://github.com/Amber-Crystal/amber/archive/v$AMBER_VERSION.tar.gz | tar xvz -C /usr/local/share/. && cd /usr/local/share/amber-$AMBER_VERSION && crystal deps && make
```

Comment:
should we look at building this like we are in the root Dockerfile instead of needing to remember to update the version number every time?

Reply:
Yeah we definitely should. I'll change that.

Reply:
@elorest I think `Amber-Crystal` should be lowercase `amber-crystal`

Reply:
Technically that's true but it used to be `Amber-Crystal` and github works with both.

Reply:
Changed that.